src/reanimated2/layoutReanimation/web/animationParser.ts

```diff
@@ -65,3 +67,33 @@ export function convertAnimationObjectToKeyframes(
 
   return keyframe;
 }
+
+export function convertTransformToString(
+  transform: NonNullable<TransformsStyle['transform']> | undefined
+) {
+  if (!transform) {
+    return '';
+  }
+
+  type RNTransformProp = (typeof transform)[number];
+
+  let transformString = '';
+
+  // @ts-ignore `transform` cannot be string because in that case
+  // we throw error in `extractTransformFromStyle`
+  transform.forEach((transformObject: RNTransformProp) => {
+    for (const [key, value] of Object.entries(transformObject)) {
+      if (key === 'reversed') {
+        continue;
+      }
+
+      if (key.indexOf('translate') < 0) {
+        transformString += `${key}(${value}) `;
+      } else {
+        transformString += `${key}(${value}px) `;
```

Comment:
What about `deg` from `rotation` property?

Reply:
I don't think you can pass `rotate` property without specifying either `deg` or `rad`. I've tried that, adding any of them works, but using just a number results in component having no rotation.